src/Altinn.Apps/KubernetesWrapper/deployment/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
Shouldnt all the deploy files be added to altinn-studio-ops instead?
we have agreed that deployment code should be in the ops repo, so this should be moved there aswell, unless there is a reason not to

Reply:
That could definitely be a possible solution. We have a follow up task in #1344 addressing this, my suggestion is to wait until this should be done before we start moving these files.

Reply:
Sounds good